app/controllers/apps/letsencrypt_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Apps::LetsencryptController < ServerBaseController
```

Comment:
`LetsEncrypt` are two words, so maybe the E should also be capitalised.

Reply:
Yep! Ok will change all file names and folder names.

Reply:
I've renamed the class names (and thus file names). I think it reduces readability and code writing for variable names and columns however. And for most tools and commands out there, LetsEncrypt is always written as "letsencrypt" and not "lets-encrypt" or "lets_encrypt".

So I'm doubting if we should not follow the community pattern too and always make it one word. Even if the "marketing name" has CamelCase.